src/test/fakes.rs

```diff
@@ -167,6 +167,11 @@ impl<'a> FakeRelease<'a> {
         self.source_file("README.md", content.as_bytes())
     }
 
+    pub(crate) fn add_owner(mut self, owner: CrateOwner) -> Self {
+        self.registry_crate_data.owners.push(owner);
```

Comment:
Hmm ... now I wonder if `owners` should be a set instead. I guess it doesn't really matter either way since we're using UPSERT.